README.md

```diff
@@ -10,7 +10,13 @@ React Native tracker for [Heap](https://heapanalytics.com).
 
 Thanks for using Heap's React Native SDK! If you're having any issues, please reach out to customer support at <support@heap.io>. For the best service, include "React Native" in the subject and your app id or customer email address in the body. Also, feel free to file a github issue or open a PR! If you do so, be sure to include a link in your request to customer support. Our engineering team checks for new PRs and github issues and tries to respond as soon as possible.
 
-## Getting Started
+
+## Getting Started Expo Managed Workflow
+```bash
+expo install @heap/react-native-heap
```

Comment:
Will this actually work for Expo based on your findings or should we tell people to use `yarn/npm install` and instructions to manually add the plugin?